src/main/resources/logback.xml

```diff
@@ -12,9 +12,11 @@
           <pattern>%date [%thread] %-5level %logger{35} - %msg%n</pattern>
         </encoder>
       </appender>
-      <root level="TRACE">
+      <root level="INFO">
         <appender-ref ref="SERVER_APPENDER" />
       </root>
+      <logger name="com.zaxxer.hikari" level="ERROR"/>
+      <logger name="HikariPool" level="ERROR"/>
```

Comment:
Does this need to be single out for ERROR-only logging?  Not even WARN would do?